src/main/java/org/jenkinsci/plugins/github/webhook/WebhookManager.java

```diff
@@ -183,7 +187,9 @@ public GHHook apply(GitHubRepositoryName name) {
         return new Predicate<GHHook>() {
             @Override
             public boolean apply(GHHook input) {
-                LOGGER.debug(format("%s {} (events: {})", format), input.getUrl(), input.getEvents());
+                if (input != null) {
```

Comment:
this method is private. No any can call it with null. And it cannot be null because it gets hook from previous step